src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Resources
+sidebar_order: 50
+description: "Learn more about browser resource performance monitoring, which allows you to see debug the performance of loading JavaScript and CSS on your frontend."
+---
+
+If you have performance monitoring enabled for your frontend, you can see how your browser resources are performing in Sentry.
+
+Starting with the [**Resources** page](#resources-page), you get a high-level overview of how your resources are doing. From there, you can drill into a specific query's [**Resource Summary** page](#resource-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
+
+<!-- TODO add gif -->
+
+## Prerequisites and Limitations
+
+The resource pages are only available for frontend JavaScript projects with performance monitoring enabled. You need to enable automatic instrumentation via the `BrowserTracing` integration for your frontend project to see resource performance data.
+
+Automatic Instrumentation for frontend JavaScript is supported for the following platforms:
+
+- [Vanilla JavaScript](/platforms/javascript/performance/)
+- [Angular](/platforms/javascript/guides/angular/performance/)
+- [Astro](/platforms/javascript/guides/astro/performance/)
+- [Ember](/platforms/javascript/guides/ember/performance/)
+- [Gatsby](/platforms/javascript/guides/gatsby/performance/)
+- [Next.js](/platforms/javascript/guides/nextjs/performance/)
+- [React](/platforms/javascript/guides/react/performance/)
+- [Remix](/platforms/javascript/guides/remix/performance/)
+- [Svelte](/platforms/javascript/guides/svelte/performance/)
+- [SvelteKit](/platforms/javascript/guides/sveltekit/performance/)
+- [Vue](/platforms/javascript/guides/vue/performance/)
+
+### Span Eligibility
+
+Sentry tries to extract metrics by looking at resource related spans.
+
+The JavaScript SDK automatically generates `resource` spans on `pageload` and `navigation` transactions using the browser's [Resource Timing API](https://developer.mozilla.org/en-US/docs/Web/API/Performance_API/Resource_timing).
+
+If you are using <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation</PlatformLink>, resource monitoring should work without any configuration. If you've manually instrumented Sentry, you'll need to make sure that your spans conform to our standards for the best experience:
```

Comment:
```suggestion
If you are using <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation</PlatformLink>, resource monitoring should work without any configuration.

If you've manually instrumented Sentry, you'll need to make sure that your spans conform to our standards for the best experience:
```

Let's make it a separate paragraph since the following list is only relevant for manual instrumentation.